.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -29,12 +29,86 @@ pull_test_dockers_arm64:
   variables:
     ARCH: arm64
 
+.wait_for_instances:
+  - |
+    COUNTER=0
+    while [[ $(aws ec2 describe-instances --filters 'Name=tag:team,Values=ebpf-platform' 'Name=tag:managed-by,Values=pulumi' 'Name=instance-state-name,Values=running' "Name=tag:pipeline-id,Values=${CI_PIPELINE_ID}" --output text --query 'Reservations[*].Instances[*].InstanceId' | wc -l ) != "2" && $COUNTER -le 40 ]]; do COUNTER=$[$COUNTER +1]; echo "[${COUNTER}] Waiting for instances"; sleep 30; done
```

Comment:
It seems like `aws ec2 describe-instances --filters 'Name=tag:team,Values=ebpf-platform' 'Name=tag:managed-by,Values=pulumi' 'Name=instance-state-name,Values=running' "Name=tag:pipeline-id,Values=${CI_PIPELINE_ID}"` is used a lot, could that be extracted to make this easier to read?

Reply:
maybe even just the `--filters` argument value